.github/workflows/build-and-publish.yml

```diff
@@ -59,3 +59,23 @@ jobs:
         github_actor: ${{ secrets.LNB_ACTOR }}
         github_token: ${{ secrets.LNB_TOKEN }}
 
+  test-build:
+    name: "Test published image"
+    needs: build
```

Comment:
Dependent on build & publish successfully finishing. Ensures the platform-core:tag will always be present for the az-bvt tests.